### PR TITLE
food doesnt come with plate

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
not in sprite or as trash when eaten

## Why / Balance
- not part of recipe
- sliced foods basically dupe recyclable mats because you can make a cake cut it up and then you got 5 small plates, 2.5 glass
- preparation for plate actually being good

## Technical details
no

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
:cl:
- tweak: Eating some foods no longer creates a plate out of thin air.

### CODE REVIEW

**Comment by Partmedia** [comment]
I do kind of actually like that food comes on plates, can we defer this until #19476 is ready?

**Comment by deltanedas** [comment]
they go hand in hand

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pie.yml [hunk 1]
```diff
@@ -8,8 +8,6 @@
   - type: FlavorProfile
     flavors:
       - sweet
-  - type: Food
-    trash: FoodPlateTin
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pie.yml:12]: shouldn't pies still create tins when eaten?

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pie.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pie.yml:12]: pretty sure they still do i just removed that for tarts to not?

Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pie.yml:12]: cream pie still gives tin the others never visually had a tin

**Review by EmoGarbage404 — APPROVED** [review]

**Comment by EmoGarbage404** [comment]
missed opportunity to add a plate to every single recipe

**Comment by deltanedas** [comment]
steak famously cooked with a plate

**Pull request merged**